### DISCUSSION TITLE
Any support for PLY splattering files?

### DISCUSSION THREAD

**Discussion by laultman** [StereoKit/StereoKit#945] [Q&A]
@maluoi Do you know of any support for ply types and how we might incorporate those into the HoloLens?

**Comment by maluoi** [comment]
If you mean like Gaussian Splatting, I haven't seen anyone do that with StereoKit yet! While it may be possible to implement, I would be hesitant to do it on a HoloLens. To come close in terms of performance, you might need some additional graphics features SK doesn't support well yet, like compute shaders.

**Comment by laultman** [comment]
I saw the name and a sample file, couldn't remember its real name. It is impressive on a computer screen. Just wondering... Thanks.
